python/examples/schedule/transform_a_payload_according_to_a_schedule.py

```diff
@@ -0,0 +1,120 @@
+# RUN: %PYTHON %s | FileCheck %s
+
+# A basic example of generating a payload, a schedule, and applying the latter
+# to the former. Shows how to do it from Python and from the cmd given the
+# payload and schedule are .mlir files. Run this file to see the concrete
+# schedule IR, pre-transform payload IR and transformed payload IR.
+
+import tempfile
+import subprocess
+
+from mlir.ir import Context, Location, InsertionPoint, Operation, Module
+from mlir.ir import RankedTensorType, F32Type, FloatAttr, DenseElementsAttr, UnitAttr
+from mlir.dialects import arith, func, linalg, tensor, transform
+from mlir.dialects.transform import structured
+
+
+def example_payload() -> Module:
+    """Example payload where the results of two matmuls are summed together.
```

Comment:
[nit-1] Please use symbols rather than words to describe Maths and code , e.g. :
```mlir
A = linalg.matmul()
B = linalg.matmul()
C = A + B
```
With symbols, there's less scope of ambiguity and IMO it is easier and quicker to parse. This will become particularly handy when dealing with more complex inputs. Also, this way you also encode (implicitly), that this is generating Linalg MLIR. The latter is key, isn't it?

[nit-2] Remove "Example" from the description - it adds no unique information.